src/templates/ApiEndpoint.tsx

```diff
@@ -167,13 +155,10 @@ function Params({ params, objects, object, depth = 0 }) {
         <>
             <ul className="list-none pl-0">
                 {params.map((param, index) => (
-                    <li
-                        key={index}
-                        className="py-3 border-b border-gray-accent-light dark:border-gray-accent-dark first:border-0 last:border-0"
-                    >
+                    <li key={index} className="py-3 border-b border-primary last:border-0">
                         <div className="grid" style={{ gridTemplateColumns: '40% 60%' }}>
                             <div className="flex flex-col">
-                                <span className="font-code font-semibold text-[13px] leading-7">{param.name}</span>
+                                <span className="font-code font-semibold text-[13px] leading-7">{param.name} $$</span>
```

Comment:
Why add these $$ to every param name?  

<img width="1012" height="660" alt="CleanShot 2025-09-18 at 09 52 15@2x" src="https://github.com/user-attachments/assets/be90b099-4b7d-47b1-a17d-481c18315092" />


```suggestion
                                <span className="font-code font-semibold text-[13px] leading-7">{param.name}</span>
```